src/coreclr/inc/clrconfigvalues.h

```diff
@@ -348,14 +348,8 @@ RETAIL_CONFIG_DWORD_INFO(INTERNAL_JitVNMapSelBudget, W("JitVNMapSelBudget"), 100
 #else // !(defined(TARGET_AMD64) || defined(TARGET_X86) || defined(TARGET_ARM64))
 #define EXTERNAL_FeatureSIMD_Default 0
 #endif // !(defined(TARGET_AMD64) || defined(TARGET_X86) || defined(TARGET_ARM64))
-#if defined(TARGET_AMD64) || defined(TARGET_X86)
-#define EXTERNAL_JitEnableAVX_Default 1
-#else // !(defined(TARGET_AMD64) || defined(TARGET_X86)
-#define EXTERNAL_JitEnableAVX_Default 0
-#endif // !(defined(TARGET_AMD64) || defined(TARGET_X86)
 RETAIL_CONFIG_DWORD_INFO(EXTERNAL_FeatureSIMD, W("FeatureSIMD"), EXTERNAL_FeatureSIMD_Default, "Enable SIMD intrinsics recognition in System.Numerics.dll and/or System.Numerics.Vectors.dll")
```

Comment:
We need to do something about `FeatureSIMD`. Originally it was intended to be a workaround in case the `System.Numerics.Vector*` logic had issues.

However, given how critical `SIMD` is to the ABI and even basic operations in the JIT for x64 and Arm64; I think it might be better to actually remove the public facing flag here and leave it only a compile level switch for `x86 Unix` and `Arm32` (the two targets where `FEATURE_SIMD == 0` today).

Reply:
-- If we are fine with removing flags, `EXTERNAL_EnableSSE3_4` is also legacy and unnecessary as it has the same meaning as `EnableSSE3`